src/script/calling/Participant.ts

```diff
@@ -39,6 +40,7 @@ export class Participant {
   public readonly hasPausedVideo: ko.PureComputed<boolean>;
   public readonly sharesScreen: ko.PureComputed<boolean>;
   public readonly sharesCamera: ko.PureComputed<boolean>;
+  public readonly isSwitchingVideoResolution = observable(false);
```

Comment:
maybe instead of a public property this variable could be private and we could add a method like: toggleIsSwitchingVideoResolution which would set it to true and set it to false after a second, this way you could avoid writing the setTimeouts twice in here

https://github.com/wireapp/wire-webapp/pull/18944/files#diff-815c4a1834d94a723e8cd6eb18d5608e92dd62d3fb18809251711b9239baef96R275-R279

and here:

https://github.com/wireapp/wire-webapp/pull/18944/files#diff-815c4a1834d94a723e8cd6eb18d5608e92dd62d3fb18809251711b9239baef96R293-R298

Reply:
A separate method certainly makes sense. It also makes what's happening clearer.

I will not make **isSwitchingVideoResolution** private. The property is used in the view object. All such properties are **public readonly** in your application.